### ISSUE TITLE
Playing on a private sever with 3 people, only the one I’m on makes everyone but the host time out.

### ISSUE THREAD

**Issue by Eklipse15** [AllTheMods/atm6-sky#618]
The bug consists of 30+ seconds of delay, while not timing out entirely in chat and not being able to move. This hadn’t happened the two days prior, and the island hadn’t changed much. The server shuts down when we all get off, and the island isn’t even big or advanced yet, was just installing a mob grinder. Server host has perfect connection and removed everything from my island it hopes that it would fix, and nothing changed. A flat plane with a couple chests on it and less than a bar of connection. The other player also loses connection entirely when they teleport to my island, yet there’s nothing there. What could cause this, does anyone have any ideas? We have tried everything besides making a new island. All of our connections are perfect otherwise, and I can be on the hosts island which is entirely more advanced and 100x larger.

**Comment by Eklipse15** [comment]
Lots and lots of tests and tweaking later, it was the mob fans. After placing 9 mob fans, the connections is entirely unusable and broken.

tldr: Mob Fans break the servers connection for everyone but the host
